### ISSUE TITLE
Ascii table broken with some cities

### ISSUE THREAD

**Issue by cig0** [chubin/wttr.in#192]
Hi,

Please check attached screenshot:

![screen shot 2018-05-08 at 22 51 00-compressed](https://user-images.githubusercontent.com/394089/39791615-049a3482-5313-11e8-95aa-81ca8ad4ceae.png)

I find this happening with many other cities as well.

Let me know if this is a *wego* issue instead to report the bug on their repo.
Thanks!

**Comment by slump64** [comment]
It's not about cities, it's about `Thundery` part. I guess thunder emoji(?) breaks the terminal output or font because it is animated.
![same](https://user-images.githubusercontent.com/14310552/40257053-3becd540-5adc-11e8-963e-54bc05631238.png)

**Comment by MAGICCC** [comment]
@slump64 yes, see here https://github.com/chubin/wttr.in/issues/100
But just saw that @cig0 screenshot only shows rain symbols so the drops are wide symbols as well

**Comment by chubin** [comment]
@MAGICCC now it is not rain, it is thunder (or better to say lighting)

**Comment by chubin** [comment]
@slump64 it is not an emoji, it is just a normal wide unicode character; I will fix it. Actually after I've added ja/ko support, that us wide characters, the fix is trivial

**Comment by slump64** [comment]
@chubin OFFTOPIC: pretty sure the fact that this symbol is a `normal wide unicode character` doesn't make it an `not emoji` (https://emojipedia.org/high-voltage-sign/)

**Comment by chubin** [comment]
@slump64 yes, you are right, strictly speaking it is an emoji, it would be better to write "even if it is an emoji, it does not matter, because it is just a normal wide unicode symbol and so it can be handled as such". Actually many of normal symbols are emojis, as far as I know, even such symbols as +, - and ! and so on (©️, ®️, ☯️, ™, etc. I'm not sure, but  I think that all of them are emojis). Thus, you are surely right

**Comment by chubin** [comment]
The problem was reported again: https://twitter.com/neproshennie/status/1001575302647242753

**Comment by R3l4x3** [comment]
I love this tool 🙂 but unfortunately have the same display error

![image](https://user-images.githubusercontent.com/5042379/42203607-9c0e9f3a-7e9f-11e8-80b7-67f0bb7c9e35.png)

**Comment by 07pepa** [comment]
also problems http://prntscr.com/kpdl56

**Comment by chubin** [comment]
@07pepa What was that? Chrome?

**Comment by 07pepa** [comment]
nope and i know what is causing it... it is debian in WSL

**Comment by jrcharney** [comment]
Howdy! I'm using KiTTY as my SSH program, which is a fork of Putty to access my Raspberry Pi 3 which I've set up to use 256 colors and zsh.

I decided to browse around to see what other icons are used to represent the weather when I took notice of an area that was having thunderstorms both as part of the weather report and in the forecast.

One of the things I've noticed is how the lightning bolts' appearance.

![image](https://user-images.githubusercontent.com/1954239/53291397-28b40380-3778-11e9-8277-4c9db4d4344f.png)

I was thinking as a substitute for the ASCII/Emoji lightning icons, try ASCII Art, because this is what that forecast looks like here on Github.
```
Weather report: Jackson, MS

      .-.      Thunderstorm In Vicinity
     (   ).    82..86 °F
    (___(__)   ↑ 16 mph
   ‚‘⚡‘‚⚡‚‘  9 mi
   ‚’‚’⚡’‚’   0.0 in

```

I thought about some alternatives. After a google search to [I saw this](http://www.ascii-art.de/ascii/jkl/lightning.txt) (The last example down at the bottom) and tried to minimize it. I don't think I could make it any smaller without losing some detail, but do you think you can work with something like this?

```
    ,/		 Lightning bolt (Similar to Corcoran's but smaller. Still too big.)
  .'/___,
.'__  ,'
   /,'
  /'

  
    ___       Lightning bolt (but this one would be a little too big)
   /  /_,
  /_  ,'
   /,'
  /'
  
 So I came up with this. For all the following case, the lightning bolt would be colored yellow to differentiate that from rain. which would use similar character.
  
    __       Lightning bolt (raw, no clouds or precip.)
   / /_,
  /_ ,'
   /'

  

   .-.      Thunderstorm (small cloud, no precip.)
  ( __).
 (_/ /_,)
  /_ ,'
   /'
   
_`/"".-.      Thunderstorm (small cloud, no precip. with sun)
 ,\_( __).
  /(_/ /_,)
    /_ ,'
     /'

_`/"".-.      Thunderstorm (small cloud, light rain. with sun)
 ,\_( __).
  /(_/ /_,)
   ‘/_ ,' ‘
  ‘ ‘/' ‘ ‘

_`/"".-.      Thunderstorm (small cloud, light moderate/heavy rain. with sun)
 ,\_( __).
  /(_/ /_,)
  ‚‘/_ ,'‚‘
  ‚‘,/'‚‘‚‘
	 
	 
_`/"".-.      Thundersnow (small cloud, snow. with sun)
 ,\_( __).
  /(_/ /_,)
   */_ ,'*
  * */'*
	 
    .--.		Thunderstorm (large cloud, no precip.)
 .-(  __).  
(___./ /__)
    /_ ,'
     /'
	 
    .--.		Thunderstorm (large cloud, light rain.)
 .-(  __).  
(___./ /__)
 ‘ ‘/_ ,'‘
‘ ‘  /' ‘
	 
     .--.		Thunderstorm (large cloud, with moderate/heavy rain.)
  .-(  __).  
 (___./ /__)
‚‘‚‘‚/_ ,'‚‘
‚‘‚‘, /'‚‘‚‘
	 
     .--.		Thundersnow (large cloud, snow.)
  .-(  __).  
 (___./ /__)
 * * /_ ,'*
* * * /'*
```

Might want to move the lightning bolt to the left a little bit to make it more clear. (I wish Github would change the comment textarea to use a monospace font instead of a sans-serif. I had to whip these up in my text editor.

**Comment by jrcharney** [comment]
OK, let's try to make those no-sun icons look better.

```
    .--.		Thunderstorm (large cloud, no precip., lightning more centered)
 .-( __ ).  
(__./ /___)
   /_ ,'
    /'
	
     .--.		Thunderstorm (large cloud, no precip., lightning more centered)
  .-( __ ).  
 (__./ /___)
 ‘ ‘/_ ,' ‘
‘ ‘ ‘/'‘ ‘

     .--.		Thunderstorm (large cloud, heavy/moderate rain, lightning more centered)
  .-( __ ).  
 (__./ /___)
‚‘‚‘/_ ,'‚‘
‚‘‚‘,/'‚‘‚‘

     .--.		Thundersnow (large cloud, snow, lightning more centered)
  .-( __ ).  
 (__./ /___)
 * */_ ,'* *
* * */' * *
```

**Comment by jrcharney** [comment]
Now that I think of it, I should have submitted these comments to schachmat/wego

**Comment by chubin** [comment]
@jrcharney Sure, you can do it, and so improve the both projects; we can use your improvements independently anyway; wego and wttr.in diverged many years ago (in 2016).

Maybe we should have create a non-unicode/non-emoji version, where only real ASCII symbols would be used? There we could the proposed by @jrcharney fixes.

Another option: we could make an option by means of which actual width of the lightning emoji could be chosen. The problem is (as you probably understand) that the with of the lighting emoji/sign varies in different terminals/browsers, so when we fix it for ones it does not work the others and other way around. But such an option could be a workaround

**Comment by nixpanic** [comment]
![wttr in amsterdam_lightning-in-firefox](https://user-images.githubusercontent.com/400257/75439290-cfb61700-5959-11ea-9d29-86a07c3cbdde.png)

The problem is currently visible in Firefox 73 (on Fedora 31). No display issue on the XFCE Terminal.

**Comment by chubin** [comment]
@nixpanic Niels, thank you for reporting. We should fix it some day
